cli/src/commands/apply/resources.ts

```diff
@@ -3,6 +3,7 @@ import * as path from 'path';
 import { Command, flags } from '@oclif/command';
 import { safeLoadAll } from 'js-yaml';
 import { ResourceGroupInput, uploadResourceGroup } from '../../client';
+import buildStitchedSchema from '../../utils/read-schema-files';
```

Comment:
Should probably reflect the new file name here

```suggestion
import readSchemaFiles from '../../utils/read-schema-files';
```

I also think `concatenateSchemaFiles` would be a more descriptive name for the file